gems/quilt_rails/lib/quilt_rails/performance/connection.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module Quilt
+  module Performance
+    class Connection
```

Comment:
In general, when they log out these model-style classes, do they present useful information? I don't know how the default logging of Rails class instances work.

Reply:
It just says the class name and a uuid type thing. I think there's like a toString / inspect type method you can implement to show more details, I'll look into it.